src/graphql/builder.tsx

```diff
@@ -44,6 +44,7 @@ const builder = new SchemaBuilder<PothosTypes>({
   ],
   prisma: {
     client: prisma,
+    dmmf: getDatamodel(),
```

Comment:
I believe this is safe to add regardless of deployment runtime. cc @hayes correct me if I'm wrong